src/foam/u2/ActionView.js

```diff
@@ -63,6 +63,7 @@ foam.CLASS({
   properties: [
     'data',
     'action',
+    [ 'nodeName', 'button'],
```

Comment:
Sorry, what? When did that happen?

More importantly, why? Isn't a property you can override more flexible?